src/Besql/Bit.Besql/wwwroot/bit-besql.js

```
@@ -1,66 +1,48 @@
-var BitBesql = BitBesql || {};
+﻿var BitBesql = BitBesql || {};
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)